packages/workbox-broadcast-cache-update/broadcastUpdate.mjs

```diff
@@ -55,6 +56,18 @@ import './_version.mjs';
  * @memberof workbox.broadcastUpdate
  */
 const broadcastUpdate = (channel, cacheName, url, source) => {
+  // There are browsers which support service workers but don't support the
+  // Broadcast Channel API.
+  // See https://github.com/GoogleChrome/workbox/issues/1304
+  if (!('BroadcastChannel' in self)) {
```

Comment:
This should be checking if the channel exists as well

Reply:
This still holds true for the PR I think - we should check if the channel exists and return if not.